### ISSUE TITLE
Error CS0246: The type or namespace name 'MauiUIApplicationDelegate' could not be found (are you missing a using directive or an assembly reference?) (CS0246)

### ISSUE THREAD

**Issue by ssbssb-cmd** [dotnet/maui#20649]
### Description

I have upgraded the .net and it is still not working.. please can you help? My project is not compiling.

Error CS0246: The type or namespace name 'MauiUIApplicationDelegate' could not be found (are you missing a using directive or an assembly reference?)

I have already deleted the bin and obj folders, clean solution and restarted my Mac, but still getting this error.

```
dotnet --version
8.0.201
```

I have run sudo dotnet workload update and still it is showing the 8.0.200 one

visual studio Mac 17.6.9 build 415

```
Installed Workload Id      Manifest Version      Installation Source
maui-ios                   8.0.6/8.0.100         SDK 8.0.200   
```

I see this in the build output, not sure why it is selecting 8.0.100
```
Target _CheckForNETCoreSdkIsPreview:
    /usr/local/share/dotnet/sdk/8.0.100-rc.2.23502.2/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.RuntimeIdentifierInference.targets(311,5): message NETSDK1057: You are using a preview version of .NET. See: https://aka.ms/dotnet-support-policy
```

### Version with bug

Unknown/Other

### Is this a regression from previous behavior?

Yes, this used to work in .NET MAUI

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Did you find any workaround?

No

**Comment by ssbssb-cmd** [comment]
I've added a global.json file in my project root and it seems to work

```
{
  "sdk": {
    "version": "8.0.201",
    "rollForward": "disable"
  }
}
```

**Comment by drasticactions** [comment]
It's this issue: https://github.com/dotnet/maui/issues/20600

Which should be addressed in 8.0.201. If anyone is still having issues with this after updating to 8.0.201, then update this above issue.
